src/towncrier/test/test_create.py

```diff
@@ -172,17 +172,39 @@ def test_create_orphan_fragment(self):
         """
         runner = CliRunner()
 
+        fragments = []
         with runner.isolated_filesystem():
```

Comment:
Could this be `@with_isolated_runner` instead?

Reply:
Sure, I was just copying the style of the tests surrounding when I wrote this originally, but I'll switch to that decorator.